src/index.js

```
@@ -1 +1,37 @@
+/* eslint-disable no-console */
 'use strict';
+
+import 'dotenv/config';
+import express from 'express';
+import { authRouter } from './routes/auth.route.js';
+import cors from 'cors';
+import { userRouter } from './routes/user.route.js';
+import { errorMiddleware } from './middlewares/errorMiddleware.js';
+import cookieParser from 'cookie-parser';
+
+const PORT = process.env.PORT || 3005;
+
+const app = express();
+
+app.use(express.json());
+app.use(cookieParser());
+
+app.use(
+  cors({
+    origin: process.env.CLIENT_HOST,
+    credentials: true,
+  }),
+);
+
+app.use(authRouter);
+app.use('/users', userRouter);
+
+app.get('/', (req, res) => {
+  res.send('Hello');
+});
+
```

The app currently has no 404 catch-all handler. The task requires returning/rendering a 404 page for all non-existing routes. Add a handler after your routes and before the error middleware, for example:
```
app.use((req, res) => res.status(404).send({ message: 'Not found' }));
```
Place it before the error middleware so unknown routes produce 404 rather than falling through. (Checklist: 404 for all other pages)